### PR TITLE
Add get-mapping-fields command for Elasticsearch schema

### PR DESCRIPTION
**Pull request by sebadoh**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add get-mapping-fields command to retrieve the elasticsearch schema 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0
- [X] 6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 1]
```diff
@@ -594,6 +593,27 @@ def fetch_incidents():
     demisto.incidents(incidents)
 
 
+def parse_subtree(my_map):
```
Comment (hod-alpert) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:596]: docstring and type hints

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 2]
```diff
@@ -603,6 +623,8 @@ def main():
             fetch_incidents()
         elif demisto.command() in ['search', 'es-search']:
             search_command()
+        elif demisto.command() == 'get-mapping-fields':
```
Comment (hod-alpert) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:626]: Add this command to the yml file too

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 3]
```diff
@@ -594,6 +593,27 @@ def fetch_incidents():
...
+    res = {}
+    for k in my_map:
+        if 'properties' in my_map[k]:
+            res[k] = parse_subtree(my_map[k]['properties'])
+        else:
+            res[k] = "type: " + my_map[k].get('type', "")
+    return res
+
+
+def get_mapping_fields_command():
```
Comment (hod-alpert) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:606]: Docstring and type hints

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py (hunk 2 above)
Comment (sebadoh) [reply Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:631]: 'get-mapping-fields' doesn't need to be declared in the yml. The command is like fetch-incidents - built in and called by the platform, not users. You can double check - Aviad and Slavik will be able to confirm for him.

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py (hunk 1 above)
Comment (sebadoh) [reply Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:596]: Done

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py (hunk 3 above)
Comment (sebadoh) [reply Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:611]: Done

**Comment by hod-alpert** [comment]
@sebadoh 
You are right when saying it is not mandatory to add the `get-mapping-fields` command in the yml, but it is still best practice in order to debug this command.

However, what is the use case for which you are adding this command alone?

It's usually used for mirroring, but you're not implementing the other needed commands for that.

Can you elaborate about the use case for which you're adding this for?

**Comment by sebadoh** [comment]
Added unittest and docstrings.

**Pull request merged**